.github/workflows/rust-gpu.yml

```
@@ -9,7 +9,7 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  test-gpu:
+  unit-tests-gpu:
```

I'd rather stick with the file name
```suggestion
  rust-gpu-tests:
```

---

Done.